### DISCUSSION TITLE
Dropdown with 24 options

### DISCUSSION THREAD

**Discussion by michaelschulte** [rubenarslan/formr.org#573] [General]
Hi, 

I am trying to implement a dropdown menu (select_one) with 24 options, but get only 14 shown ... is there an option limit I can update somewhere? 

Thanks!
Michael

**Comment by dgerdesmann** [comment]
Probably resolved long ago, but yes, you can only specify 12 choices in the survey spreadsheet. If you need more, add a "choices" spreadsheet (see the "Choices Spreadsheet" in the documentation for examples). Then reference the choices list name in your survey item "type" column, e.g. "select_or_add_one my_custom_list". Leave the choice columns in the survey spreadsheet blank.
